config.lua

```diff
@@ -52,38 +52,38 @@ Config.TextFr = {
 
 
 Config.TextEn = {
-	start         = "The BanList and history has been loaded successfully.",
-	starterror    = "ERROR: The BanList and history has not been loaded new try.",
-	banlistloaded = "The BanList has been loaded successfully.",
-	historyloaded = "The BanListHistory has been loaded successfully.",
-	loaderror     = "ERROR: The BanList was not loaded.",
-	forcontinu    = " days. To continue entering /sqlreason (Ban reason)",
-	noreason      = "unknown reason",
-	during        = " during : ",
-	noresult      = "There are not as many results!",
-	isban         = " was ban",
-	isunban       = " was unban",
-	invalidsteam  =  "You should open steam",
-	invalidid     = "Player ID incorrect",
-	invalidname   = "The name is not valid",
-	invalidtime   = "Bad ban duration",
-	yourban       = "You have been ban for : ",
-	yourpermban   = "You have been ban permanent for : ",
-	youban        = "You have ban : ",
-	forr          = " days. For : ",
-	permban       = " permanently for : ",
-	timeleft      = ". Time remains : ",
-	toomanyresult = "Too many results, be sure to be more precise.",
-	day           = " Days ",
-	hour          = " Hours ",
-	minute        = " Minutes ",
+	start         = "BanList and BanListHistory loaded successfully.",
+	starterror    = "ERROR: BanList and BanListHistory failed to load, please retry.",
+	banlistloaded = "BanList loaded successfully.",
+	historyloaded = "BanListHistory loaded successfully.",
+	loaderror     = "ERROR: The BanList failed to load.",
+	forcontinu    = " days. To continue, execute /sqlreason [reason]",
+	noreason      = "No reason provided.",
+	during        = " during: ",
+	noresult      = "No results found.",
+	isban         = " was banned",
+	isunban       = " was unbanned",
+	invalidsteam  = "Steam is required to join this server.",
+	invalidid     = "Player ID not found",
+	invalidname   = "The specified name is not valid",
+	invalidtime   = "Invalid ban duration",
+	yourban       = "You have been banned for: ",
+	yourpermban   = "You have been permanently banned for: ",
+	youban        = "You are banned from this server for: ",
+	forr          = " days. For: ",
+	permban       = " permanently for: ",
+	timeleft      = ". Time remaining: ",
+	toomanyresult = "Too many results, be more specific to shorten the results.",
+	day           = " days ",
+	hour          = " hours ",
+	minute        = " minutes ",
 	by            = "by",
-	ban           = "Ban one online player",
-	banoff        = "Ban one offline player",
-	dayhelp       = "Number of day",
-	reason        = "Reason of ban",
-	history       = "Show all ban of a player",
-	reload        = "Reload BanList and BanListHistory",
-	unban         = "Remove one ban from the list",
-	steamname     = "(Steam Name)",
+	ban           = "Ban a player",
+	banoff        = "Manually ban a (offline) player",

```

Comment:
Manually are not need to be write.
It's just an in game help to say what the command do.
"Ban an offline player" Will be better no?